ext/standard/tests/misc/gh14774.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+GH-14774 time_sleep_until overflow
+--SKIPIF--
+<?php
+if (PHP_INT_SIZE != 8) die("skip this test is for 64bit platform only");
+?>
+--FILE--
+<?php
```

Comment:
Please also test positive and negative INF, positive NAN, and a large negative value (e.g. -10e300) for completeness.
You can use a foreach loop over an array containing these values to avoid duplicating code and keeping the test clean.